autoshop/core/site_processor.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+def setting_and_site_context(request):
```

Comment:
rename the file to `context_processors` and rename this method to `site_processor`